gravitee-apim-console-webui/src/management/api/api-ng-navigation/api-ng-navigation.component.ts

```diff
@@ -56,16 +74,172 @@ export class ApiNgNavigationComponent implements OnInit {
+          type: 'success',
+          action: {
+            btnText: 'Review changes',
+            onClick: () => {
+              this.matDialog
+                .open<ApiReviewDialogComponent, ApiReviewDialogData, ApiReviewDialogResult>(ApiReviewDialogComponent, {
+                  data: {
+                    apiId: api.id,
+                  },
+                  role: 'alertdialog',
+                  id: 'gioApiReviewDialog',
+                  width: GIO_DIALOG_WIDTH.MEDIUM,
+                })
+                .afterClosed()
+                .pipe(takeUntil(this.unsubscribe$))
+                .subscribe();
+            },
+          },
+        });
+      }
+      if (isApiReviewer && this.constants.env.settings?.apiReview?.enabled && api.workflowState === 'REQUEST_FOR_CHANGES') {
+        banners.push({
+          title: 'As an API reviewer, you have rejected the changes made on this API.',
+          type: 'warning',
+          action: {
+            btnText: 'Review changes',
+            onClick: () => {
+              this.matDialog
+                .open<ApiReviewDialogComponent, ApiReviewDialogData, ApiReviewDialogResult>(ApiReviewDialogComponent, {
+                  data: {
+                    apiId: api.id,
+                  },
+                  role: 'alertdialog',
+                  id: 'gioApiReviewDialog',
+                  width: GIO_DIALOG_WIDTH.MEDIUM,
+                })
+                .afterClosed()
+                .pipe(takeUntil(this.unsubscribe$))
+                .subscribe();
+            },
+          },
+        });
+      }
+
+      // Only for not API reviewer
+      if (!isApiReviewer && this.constants.env.settings?.apiReview?.enabled && api.workflowState === 'IN_REVIEW') {
+        // not 'api-reviews-u'
+        banners.push({
+          title: 'The API reviewer has been asked to review the changes.',
+          type: 'info',
+        });
+      }
+      if (!isApiReviewer && this.constants.env.settings?.apiReview?.enabled && api.workflowState === 'REVIEW_OK') {
+        // not 'api-reviews-u'
+        banners.push({
+          title: 'The API reviewer has accepted the changes made on this API.',
+          type: 'success',
+        });
+      }
+      if (!isApiReviewer && this.constants.env.settings?.apiReview?.enabled && api.workflowState === 'REQUEST_FOR_CHANGES') {
+        // not 'api-reviews-u'
+        banners.push({
+          title: 'The API reviewer has asked for changes to be made on this API.',
+          type: 'warning',
+        });
+      }
+
+      // Other banners
+      if (this.constants.env.settings?.apiReview?.enabled && api.workflowState === 'DRAFT') {
+        banners.push({
+          title: 'This API is a draft.',
+          type: 'info',
+        });
+      }
+
+      const apiReviewIsOKOrNotNeeded = (this.constants.env.settings?.apiReview?.enabled && api.workflowState === 'REVIEW_OK') || true;
```

Comment:
When is it possible for this to be false?
Maybe you mean
```suggestion
      const apiReviewIsOKOrNotNeeded = (this.constants.env.settings?.apiReview?.enabled && api.workflowState === 'REVIEW_OK') || !this.constants.env.settings?.apiReview?.enabled;
```

Reply:
hum yes good catch.